gpkit/nomials/substitution.py

```diff
@@ -125,11 +125,20 @@ def substitution(nomial, substitutions, val=None):
             if len(varlocs_[var]) == 0:
                 del varlocs_[var]
             if isinstance(sub, Numbers):
+                if isinstance(sub, Quantity):
```

Comment:
Sounds good! We'll be assuming that anything with units also has a `to` method, though.